native/src/boot/bootimg.cpp

```diff
@@ -620,6 +649,17 @@ void repack(const char *src_img, const char *out_img, bool skip_comp) {
     hdr->ramdisk_size() = 0;
     hdr->second_size() = 0;
     hdr->dtb_size() = 0;
+    hdr->bootconfig_size() = 0;
+    hdr->vendor_ramdisk_table_size() = 0;
+    hdr->vendor_ramdisk_table_entry_num() = 0;
+
+    std::vector<std::tuple<std::unique_ptr<vendor_ramdisk_table_entry_v4>, format_t>> new_ramdisk_table_entries;
```

Comment:
for `new_ramdisk_table_entries`, I think we can use `std::vector<std::tuple<vendor_ramdisk_table_entry_v4, format_t>>` because we should have a default copy constructor for `vendor_ramdisk_table_entry_v4`

Reply:
done